Tools/machines/lxplus-cern/spack.yaml

```diff
@@ -0,0 +1,56 @@
+spack:
+# CUDA
+  definitions:
+  - cuda: []
+  - cuda: [cuda, blaspp +cuda]
+    when: env.get("SPACK_STACK_USE_CUDA", "") == "1"
+
+# basic Python dependencies for WarpX Python PICMI bindings
+  - python: []
+  - python: [python, py-pip]
+    when: env.get("SPACK_STACK_USE_PYTHON", "") == "1"
+  specs:
+  - ncurses+symlinks
+  - adios2
+  - blaspp
+  - ccache
+  - cmake
+  - fftw
+  - hdf5
+  - lapackpp
+  - mpi
+  - openpmd-api
+  - pkgconfig
+  - $cuda
+  - $python
+
+  packages:
+    all:
+      target: [x86_64]
```

Comment:
```suggestion
      # since lxplus has multiple CPU micro-architectures, we do not tune for a specific one
      # in the future, if highest on-CPU performance is needed, we should not limit this to generic x86_64,
      # otherwise the compiler will not use recent vector operations, among other recent CPU hardware features.
      target: [x86_64]
```